code/game/machinery/doors/windowdoor.dm

```diff
@@ -252,6 +252,19 @@
 	health = 300.0 //Stronger doors for prison (regular window door health is 200)
 
 
+/obj/machinery/door/window/briefing
+	name = "reinforced safety windoor"
+	desc = "An extremely strong windoor, capable of stopping the fiercest of riots."
+	icon_state = "leftsecure"
+	base_state = "leftsecure"
+	req_access = list(ACCESS_MARINE_BRIDGE)
+	health = 1000
+
+
+/obj/machinery/door/window/briefing/get_projectile_hit_chance(obj/item/projectile/P)
+	return TRUE
```

Comment:
this should serve no purpose as windoors don't account for energy projectiles (yet), if I recall correctly.